site/content/docs/tcp-checks/_index.md

```diff
@@ -11,17 +11,19 @@ slug: /
 ---
 
 > **Early Access Feature:**
-> This feature is currently in beta. To enable it in the UI, go to the [Labs section](https://app.checklyhq.com/settings/account/labs) in your account settings and toggle on TCP checks. Please note that support for Monitoring as Code (MaC) via the CLI, Terraform, and Pulumi providers is currently is still in development.
+> This feature is currently in beta and available only via the UI. Support for Monitoring as Code (MaC) through the CLI, Terraform, and Pulumi providers is still in development.
 
 ## Overview
 
-A TCP check establishes a connection to a specified hostname or IP address and port to verify responsiveness. These checks are ideal for monitoring non-HTTP services critical to your infrastructure. Here are a few common use cases:
+A TCP check establishes a connection to a specified hostname or IP address and port to verify responsiveness. These checks are ideal for monitoring non-HTTP services critical to your infrastructure. Here are a few example use cases:
 
-* **Mail servers** (e.g. `mail.example.org:993`): Ensure your mail server is available and can accept incoming IMAPS connections.
+* Mail servers (e.g. `mail.example.org:993`): Use TCP checks to ensure your mail server is online and processing requests efficiently. For example, set an assertion on the response time to confirm the server accepts IMAPS connections without delays. This helps you spot slowdowns and provide reliable email services for your users.
 
-* **Databases** (e.g. `database.example.org:3306`): Verify that your database server is online and able to accept connections.
+* Databases (e.g. `database.example.org:3306`): Check that your database is online and accepting connections. To ensure that queries are processed as expected, you can additionally send a simple query like `SELECT 1` as part of the request and confirm the response matches what you’d expect via assertions.
```

Comment:
To be able to send the `SELECT 1` to the server, the customer would need to implement the [MySQL line protocol](https://dev.mysql.com/doc/dev/mysql-server/8.4.3/page_protocol_connection_phase.html) within the payload. I don't think that's a feasible example at the moment.